File: 1 C/problem set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get input(height)
    int height;
    while (1)
    {
        height = get_int("Height:");
        if (height >0 && height <9)
        {
            break;
        }
    }

    //each line
    for (int i = 0; i < height; i++)
    {
        // " "part
        for (int j = 0; j < (height - i - 1); j++)
        {
            printf(" ");
        }
        // "#" part
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: 6 Python/problem set 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while 1:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

for i in range(h):
    # left
    print(" " * (h-i-1) + "#" * (i + 1), end=" " * 2)
    # right
    print("#" * (i + 1))